src/test/java/com/puppycrawl/tools/checkstyle/api/LineColumnTest.java

```diff
@@ -0,0 +1,54 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2015 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.puppycrawl.tools.checkstyle.api;
+
+import nl.jqno.equalsverifier.EqualsVerifier;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class LineColumnTest {
+
+    @Test
+    public void testCompareToBothEqual() {
+        assertEquals(0, new LineColumn(0, 0).compareTo(new LineColumn(0, 0)));
+    }
+
+    @Test
+    public void testCompareToFirstLarger() {
+        LineColumn lineColumn = new LineColumn(0, 0);
+
+        assertEquals(1, new LineColumn(1, 0).compareTo(lineColumn));
+        assertEquals(1, new LineColumn(0, 1).compareTo(lineColumn));
+    }
+
+    @Test
+    public void testCompareToFirstSmaller() {
+        Comparable<LineColumn> lineColumn = new LineColumn(0, 0);
```

Comment:
To achieve 100% code coverage method had to be invoked also through interface to cover bridge method: http://codeidol.com/java/javagenerics/Comparison-and-Bounds/Bridges/